### ISSUE TITLE
Foundry v10 Support

### ISSUE THREAD

**Issue by bluefroguk1** [mouse0270/midi-challenge-mode-armor#3]
Hi 
When running FVTT v10 with Midi-QOL installed, i get the following compatibility errors:

[midi-qol] Metadata validation failed for module "midi-qol": Module Model Validation Errors
[Module.packs]: Model Validation Errors
The Compendium pack "midiqol-sample-items" of the "Item" type must declare the "system" upon which it depends.

setup.js:145 [midi-qol] The module "midi-qol" contains dependencies using "name" which is deprecated in favor of "id"

setup.js:145 [midi-qol] The module "midi-qol" contains "dependencies" which is deprecated in favor of "relationships.requires"

setup.js:145 [midi-qol] module "midi-qol" contains "system" which is deprecated in favor of "relationships.systems"

**Comment by mouse0270** [comment]
I will have to update it, which is fine. I can do this tomorrow.

**Comment by p4535992** [comment]
This module is so small that I would integrate it directly on the "TidySheet" module as a feature.

**Comment by mouse0270** [comment]
The problem is, this isn't specific to TidySheet I just added support for TidySheet cause well it's really popular.

However if you'd like it implemented on TidySheet, go there and ask.

Also I don't even use the module as this isn't a system I play with, I just made it at the request of someone else.

**Comment by p4535992** [comment]
A ok you are right is working on the standard sheet too , me stupid, no hurry take your time.

**Comment by mouse0270** [comment]
Well, technically the module works fine on v10. The issue reported here is actually MIDI, not this module. I've already fixed the v10 issue in v1.0.3 I just have to push it out.

But honestly, this should be ported straight into MIDI - QOL. Cause you are right, it's super small. I can't remember my conversation with the developer at the time and if there was a reason it was released as a separate module.
